packages/aws-cdk-lib/aws-rds/test/cluster.test.ts

```diff
@@ -419,6 +440,240 @@ describe('cluster new api', () => {
       });
     });
 
+    describe('limitless database', () => {
```

Comment:
Lotsa new tests for a simple feature? Copy/paste or GenAI?

I'd rather turn existing tests into data-driven tests for both ways of passing props.

```ts
test.each([
  { clusterScailabilityType: ClusterScailabilityType.LIMITLESS },
  { clusterScalabilityType: ClusterScalabilityType.LIMITLESS },
])('my test with input %p, (partialProps) => {
  // ...
});
```